tests/API/CPE/dict/test_api_cpe_dict.c

```diff
@@ -198,14 +198,8 @@ int main(int argc, char **argv)
 		name = cpe_name_new(argv[4]);
 
 		ret_val_1 = cpe_name_match_dict(name, dict_model);
-		ret_val_2 = cpe_name_match_dict_str(argv[4], dict_model);
```

Comment:
@yuumasato Any chance we could drop the definition of [`ret_val_2`](https://github.com/OpenSCAP/openscap/blob/fcfe3f10b10c016760efae168789a3b0ae1f2fd0/tests/API/CPE/dict/test_api_cpe_dict.c#L61)? My build reports it as being unused now, which it is. :)

```c
[235/321] Building C object tests/API/CPE/dict/CMakeFiles/test_api_cpe_dict.dir/test_api_cpe_dict.c.o
../tests/API/CPE/dict/test_api_cpe_dict.c: In function ‘main’:
../tests/API/CPE/dict/test_api_cpe_dict.c:61:26: warning: unused variable ‘ret_val_2’ [-Wunused-variable]
  bool ret_val_1 = false, ret_val_2 = false;
                          ^~~~~~~~~
```